x-pack/plugins/task_manager/server/test_utils/index.ts

```diff
@@ -11,18 +11,6 @@
 // Caching this here to avoid setTimeout mocking affecting our tests.
 const nativeTimeout = setTimeout;
 
-/**
- * Creates a mock task manager Logger.
- */
-export function mockLogger() {
```

Comment:
rather than delete this method, I think we could leave all the call sites alone, and replace the body with 

```
import { loggingSystemMock } from 'src/core/server/mocks';

...

export function mockLogger() {
   return loggingSystemMock.createLogger()
}
```

I ran into something similar recently, and have gone that route.  I also needed the type of `MockedLogger` that `createLogger()` returns, but it wasn't easily available, so use the following to get the type where I needed it:

```
type MockedLogger = ReturnType<typeof loggingSystemMock['createLogger']>;
```

I really only needed this so I could do some `expect()`'s on the logger calls, if you don't need that, I think you can use `loggingSystemMock.create().get()` - one returns the mock signatures, the other doesn't.